OpenSim/Actuators/VandenBogert2011Muscle.h

```diff
@@ -0,0 +1,568 @@
+        (normalized to maxIsometricForce), b (s/m)*/
+        void setDampingCoefficient(double dampingCoefficient);
+        /** @returns the  damping coefficient of damper parallel to the fiber
+        (normalized to maxIsometricForce), b (s/m)*/
+        double getDampingCoefficient() const;
+
+        /** @param the slack length of the parallel elastic element, divided by
+        fiber optimal length, L_{slack,fiber} (dimensionless)*/
+        void setNormFiberSlackLength(double normFiberSlackLength);
+        /** @returns the slack length of the parallel elastic element, divided
+        by fiber optimal length, L_{slack,fiber} (dimensionless)*/
+        double getNormFiberSlackLength() const;
+
+        /** @param the activation time constant, T_{act} (s)*/
+        void setActivTimeConstant(double activTimeConstant);
+        /** @returns the activation time constant, T_{act} (s)*/
+        double getActivTimeConstant() const;
+
+        /** @param the deactivation time constant, T_{deact} (s)*/
+        void setDeactivTimeConstant(double deactivTimeConstant);
+        /** @returns the deactivation time constant, T_{deact} (s)*/
+        double getDeactivTimeConstant() const;
+
+        /** @param the pennation angel at optimal fiber length,
+        phi_{opt} (rad)*/
+        void setPennAtOptFiberLength(double pennAtOptFiberLength);
+        /** @returns the pennation angel at optimal fiber length,
+        phi_{opt} (rad)*/
+        double getPennAtOptFiberLength() const;
+
+        /** @param the default muscle activation, (dimensionless)*/
+        void setDefaultActivation(double defaultActivation);
+        /** @returns the default muscle activation, (dimensionless)*/
+        double getDefaultActivation() const;
+
+        /** @param the default muscle fiber length, ()*/
+        void setDefaultFiberLength(double fiberLength);
+        /** @returns the default muscle fiber length,, ()*/
+        double getDefaultFiberLength() const;
+
+
+        /**@param s The state of the system.
+        @param the muscle activation, (dimensionless)*/
+        void setActivation(SimTK::State& s,double activation) const override;
+        /**@param s The state of the system.
+        @param fiberLength the muscle fiber length, ()*/
+        void setFiberLength(SimTK::State& s,double fiberLength) const;
+
+        /* Calculate the muscle implicit residual.  Returns a state Vec2
+        containing the force residual and activation residual.  When these
+        value are equal to zero, the muscle equations are "balanced" and the
+         muscle  is muscle is in a valid state.
+                @param s The state of the system.
+                @param projFibVelNorm_guess The muscle fiber velocity, projected
+                in line with tendon and normalized by the optimal fiber length,
+                (dimensionless)
+                @param activdot_guess The muscle activation time derivative,
+                (1/s)
```

Comment:
Doxygen will give a warning unless you document all parameters; right now, `u` is not documented.